packages/wallet/src/cip30.ts

```diff
@@ -401,11 +401,7 @@ const baseCip30WalletApi = (
     const unusedAddresses = await wallet.getNextUnusedAddress();
     const addresses = trackedAddresses.filter((address) => !unusedAddresses.includes(address));
 
-    if (addresses.length === 0) {
```

Comment:
A unit test would be nice

Reply:
done and squashed @mkazlauskas (since its a tiny PR)